service/routes.go

```diff
@@ -89,6 +89,15 @@ func getSchemas() *client.Schemas {
 	authconfig.ResourceMethods = []string{"GET", "POST"}
 	authconfig.PluralName = "configs"
 
+	// TestAuthConfig
+	testAuthconfig := schemas.AddType("testAuthConfig", model.TestAuthConfig{})
+	testAuthconfig.CollectionMethods = []string{"GET", "POST"}
```

Comment:
Collection methods are the available methods for the collection at /v1-auth/testlogin
Can the user actually do a GET to this URL?